pkgs/applications/gis/grass/default.nix

```
@@ -16,14 +16,13 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkg-config makeWrapper ];
-  buildInputs = [ flex bison zlib proj gdal libtiff libpng fftw sqlite
-  readline ffmpeg netcdf geos postgresql libmysqlclient blas
-  libLAS proj-datumgrid zstd wrapGAppsHook ]
-    ++ lib.optionals stdenv.isLinux [ cairo pdal wxGTK31 ]
-    ++ lib.optional stdenv.isDarwin wxmac
-    ++ (with python3Packages; [ python python-dateutil numpy ]
-      ++ lib.optional stdenv.isDarwin wxPython_4_0
-      ++ lib.optional stdenv.isLinux wxPython_4_1);
+  buildInputs = [
+    cairo flex bison zlib proj gdal libtiff libpng fftw sqlite
```

```suggestion
    cairo zlib proj gdal libtiff libpng fftw sqlite
```
Those go to nativeBuildInputs. Can we set ``strictDeps = true``?

---

Done.